cfpb/consumerfinance.gov
# Update to latest design-system 0.37.0 | Update esbuild, cypress, eslint-plugin-jsdoc, snyk

Bring in range slider from DS and updated packages with /dist/ directories (note: these aren't used in cfgov) https://github.com/cfpb/design-system/compare/v0.35.1...v0.37.0


## Changes

- Update to latest design-system 0.37.0
- Update esbuild, cypress, eslint-plugin-jsdoc, snyk
- Fix linter issues in tdp-analytics.


## How to test this PR

1. `yarn build` and visit http://localhost:8000/consumer-tools/retirement/before-you-claim/ and compare the slider, after entering your details, to that on production. The slider on http://localhost:8000/owning-a-home/explore-rates/ should be unchanged.
